whylog/tests/tests_config/test_files/test_investigation_plan_files/log_types.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This file has the same content as `whylog/tests/tests_config/test_files/log_types.yaml`. Why it is duplicated?